featurflag.ps1

```diff
@@ -0,0 +1,60 @@
+[CmdletBinding(PositionalBinding=$false)]
+param(
+  [string]$flag = $null,
+  [switch]$enable,
+  [switch]$disable,
+  [string]$hive = "RoslynDev",
+  [switch]$set,
+  [switch]$get
+)
+
+if ($null -eq $flag -or '' -eq $flag) {
+    throw "Specify a -flag to set"
+}
+
+$flagBase = "FeatureFlags\Razor\LSP"
+$slashIndex = $flag.LastIndexOf("\")
+
+if ($slashIndex -ge 0) {
+    $flagBase = $flag.Substring(0, $slashIndex)
+    $flag = $flag.Substring($slashIndex + 1) #+1 to trim the \
+}
+
+if ($set -and $get) {
+    throw "Use only one of set or get"
+}
+
+if (-not ($set -or $get)) {
+    throw "Specify one of -set or -get"
+}
+
+if ($set)
+{
+  if ($enable -and $disable) {
+    throw "Use only one of -enable or -disable"
+  }
+
+  if (-not ($enable -or $disable)) {
+    throw "Specify one of -enable or -disable"
+  }
+}
+
+. (Join-Path $PSScriptRoot "eng" "common" "tools.ps1")
```

Comment:
nit: do we need a space here? :) (maybe we do, not really PS expert either :) )

Reply:
After the dot? That's needed because this is executing the script to add functions that it defines